docs/rbac/example.tf

```diff
@@ -0,0 +1,52 @@
+# Set up project ID variable
+# Replace {{YOUR_GCP_PROJECT_ID}} with your own project id.
+variable project {
+    type    = "string"
+    default = "{{YOUR_GCP_PROJECT_ID}}"
+}
+
+# Create the Google Cloud terraform provider
+provider "google" {
+  project = "${var.project}"
+  region  = "us-central1"
```

Comment:
region and zone aren't both necessary. Suggest just using region. Also region's value should be a variable {{YOUR_PROJECT_REGION}}

Reply:
Done